detection_rules/etc/version.lock.json

```diff
@@ -910,20 +938,27 @@
     "version": 100
   },
   "14de811c-d60f-11ec-9fd7-f661ea17fbce": {
-    "min_stack_version": "8.3",
+    "min_stack_version": "8.4",
     "previous": {
       "8.2": {
         "max_allowable_version": 99,
         "rule_name": "Kubernetes User Exec into Pod",
         "sha256": "939f1dfae51e5df729029c2bf9c6cd64c211afd38624b26e0878e4e9f0623956",
         "type": "query",
         "version": 4
+      },
+      "8.3": {
+        "max_allowable_version": 199,
+        "rule_name": "Kubernetes User Exec into Pod",
+        "sha256": "a00465734be3cc8c51d1068bd7d2d6fd67cc0144a3f4b11d969411083176df00",
+        "type": "query",
+        "version": 100
       }
     },
     "rule_name": "Kubernetes User Exec into Pod",
-    "sha256": "a00465734be3cc8c51d1068bd7d2d6fd67cc0144a3f4b11d969411083176df00",
+    "sha256": "6e44fbb4e1b299bca2aad647504db0a3ca2fb5636cf587b958bca2e4464f48d1",
     "type": "query",
-    "version": 100
+    "version": 200
```

Comment:
Rule version jumped from 100 -> 200 as expected. This rule was min stacked to 8.4 from 8.3. As a result, a buffer of 100 is given to avoid version collisions.